.github/workflows/kickstart-tests.yml

```diff
@@ -88,26 +88,58 @@ jobs:
           git log --oneline -1 origin/master
           git rebase origin/master
 
-      # FIXME: build a complete boot.iso with lorax/osbuild
-      - name: Build updates.img
-        run: |
-          scripts/makeupdates
-          gzip -cd updates.img | cpio -tv
-
       - name: Check out kickstart-tests
         uses: actions/checkout@v2
         with:
           repository: rhinstaller/kickstart-tests
           path: kickstart-tests
 
       - name: Ensure http proxy is running
+        run: sudo kickstart-tests/containers/squid.sh start
+
+      # This is really fast, but does not catch file removals or dracut changes
+      # maybe this becomes opt-in via a magic comment for efficiency reasons?
```

Comment:
I think this option could be useful. Perhaps also as an easily accessible fallback when iso generating is broken for whatever reasons.